.eslintrc.js

```diff
@@ -2,31 +2,5 @@
 // SPDX-License-Identifier: Apache-2.0
 
 module.exports = {
-  env: {
-    browser: true,
-    es6: true,
-    node: true,
-  },
-  extends: [
-    "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react/jsx-runtime",
-    "plugin:storybook/recommended",
-  ],
-  overrides: [],
-  parser: "@typescript-eslint/parser",
-  parserOptions: {
-    tsconfigRootDir: "./",
-    ecmaVersion: 2021,
-    sourceType: "module",
-    ecmaFeatures: {
-      jsx: true,
-    },
-  },
-  plugins: ["react", "@typescript-eslint", "eslint-plugin-tsdoc", "formatjs"],
-  rules: {
-    "tsdoc/syntax": "warn",
-    "formatjs/no-offset": "error",
-  },
+  extends: ["semistandard", "plugins:playwright/recommended"],
```

Comment:
Does this actually enable the unawaited promises rule?

Reply:
i'm not sure, i'll take a look in a bit